### DISCUSSION TITLE
FR: Ability to add fields to lineItems the same way that we do for orders

### DISCUSSION THREAD

**Discussion by javangriff** [craftcms/commerce#2943] [Ideas]
Currently we have the ability to add statuses to lineItems which are great to allow admin users to manage orders through the CP or to communicate stock status to the customer—an item being back-ordered, for example (Yeah I read the docs). 

In addition to this functionality it would also be awesome to be able to be able to add custom fields to lineItems the same way we can for orders. This could be handy to allow additional information to be added to a line item—an expected delivery date and tracking code, for example.

Sorry if this is something that has been discussed before, I couldn't find anything doing a bit of a search.

**Comment by chasegiunta** [comment]
@lukeholder have you given this one any thought? Having even specific relational data tied to each line item would be incredibly useful.
